cmd/ui.go

```diff
@@ -146,13 +184,18 @@ func renderMultipleBars(isTTY, goBack bool, leftMax int, pbs []*pb.ProgressBar)
 	} else {
```

Comment:
Like we discussed in slack, the line above doesn't account for progressbars that span more than 1 line

Reply:
I think this should be fixed in b9a377e0. Please test and let me know.

The code is far from pretty and `renderMultipleBars` is doing way too much... I would like to eventually clean this up by removing `ProgressBar.hijack` and rendering the first line ("status bar") separately, since it's not really a progress bar like the others. But that can wait for #1007 to be merged, it's not a blocker for now.